hack/test-update-storage-objects.sh

```diff
@@ -112,8 +112,8 @@ tests=(
 examples/persistent-volume-provisioning/rbd/rbd-storage-class.yaml,storageclasses,,slow,v1beta1,v1
 )
 
-KUBE_OLD_API_VERSION="storage.k8s.io/v1beta1,extensions/v1beta1"
-KUBE_NEW_API_VERSION="storage.k8s.io/v1,extensions/v1beta1"
+KUBE_OLD_API_VERSION="networking.k8s.io/v1,storage.k8s.io/v1beta1,extensions/v1beta1"
+KUBE_NEW_API_VERSION="networking.k8s.io/v1,storage.k8s.io/v1,extensions/v1beta1"
```

Comment:
As we still write `extensions/v1beta1` objects (see discussion below), disabling `extensions/v1beta1` will break `networking.k8s.io/v1` as the former cannot be decoded from storage anymore. KUBE_API_VERSIONS is all or nothing: types+http-handler or no registered types and no http-handler for the disabled group. I wonder whether we shouldn't have a way to keep types registered when disabling group versions.